content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -282,22 +282,31 @@ Full <abbr>parameters</abbr> (key and value) are listed separately under
 
 ## Structure of the cache directory
 
+The DVC cache is a
+[content-addressable storage](https://en.wikipedia.org/wiki/Content-addressable_storage)
+(CAS), which adds a layer of indirection between code and data. It's hidden from
+the user (and from Git), and typically local. However, it can be synchronized
```

Comment:
sounds like it's a typical scenario to have _only_ local cache and remote is kinda optional

Reply:
My main purpose here is to insert the CAS and indirection concepts.

I removed the notes about how it's hidden, local and about remote storage.

Reply:
It's a bit tricky because the definition of the cache is spread in a few places... Here, in the `cache` cmd index, and in the tooltip...